agent.go

```diff
@@ -221,7 +221,7 @@ func (p *pod) controlLoop(wg *sync.WaitGroup) {
 				continue
 			}
 
-			agentLog.Infof("Read on ctl channel failed: %v", err)
+			agentLog.Errorf("Read on ctl channel failed: %v", err)
```

Comment:
Well that's not an error, it's about the proxy disconnecting, in order to end the agent.